controllers/handler/nodenetworkconfigurationpolicy_controller.go

```diff
@@ -243,8 +244,8 @@ func (r *NodeNetworkConfigurationPolicyReconciler) Reconcile(_ context.Context,
 	return ctrl.Result{}, nil
 }
 
-func (r *NodeNetworkConfigurationPolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	allPolicies := handler.TypedMapFunc[*corev1.Node, reconcile.Request](
+func (r *NodeNetworkConfigurationPolicyReconciler) allPolicies() handler.TypedMapFunc[*corev1.Node, reconcile.Request] {
+	return handler.TypedMapFunc[*corev1.Node, reconcile.Request](
```

Comment:
Why we conert all policies into a method ? there is no dependency on the reonciler